### DISCUSSION TITLE
Include `scrollbar-gutter: stable` in the Preflight styles.

### DISCUSSION THREAD

**Discussion by manojVivek** [tailwindlabs/tailwindcss#8208] [Ideas]
Ref: https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-gutter

**Comment by lukewarlow** [comment]
I agree that would be a good default to have.

In case anyone comes across this discussion and wants a solution in the meantime. I've made an npm package with a plugin for scrollbar gutter (along with width and color) utilities. See https://github.com/lukewarlow/tailwind-scrollbar-utilities

While this doesn't apply by default it does at least give you the utilities you need to set this property.
